vernac/mltop.ml

```diff
@@ -334,23 +333,38 @@ let _ =
 
 (* Liboject entries of declared ML Modules *)
 
+(* Digest of module used to compile the file *)
+type ml_module_digest =
```

Comment:
Why not use an `option` type?

Reply:
This could also contain native/byte dependent digest so I'd like it to be extensible; also, I find most option types in Coq hard to read.